vizro-core/examples/scratch_dev/app.py

```diff
@@ -1,113 +1,49 @@
-"""Dev app to try things out."""
-
+import plotly.graph_objs as go
+import plotly.io as pio
 import vizro.models as vm
 import vizro.plotly.express as px
 from vizro import Vizro
-from vizro.figures import kpi_card
 from vizro.models.types import capture
-from vizro.tables import dash_ag_grid, dash_data_table
-
-df = px.data.iris()
-
-
-# Graph
-@capture("graph")
-def my_graph_figure(data_frame, **kwargs):
-    """My custom figure."""
-    return px.scatter(data_frame, **kwargs)
-
-
-class MyGraph(vm.Graph):
-    """My custom class."""
-
-    def build(self):
-        """Custom build."""
-        graph_build_obj = super().build()
-        # DO SOMETHING
-        return graph_build_obj
-
-
-# Table
-@capture("table")
-def my_table_figure(data_frame, **kwargs):
-    """My custom figure."""
-    return dash_data_table(data_frame, **kwargs)()
-
-
-class MyTable(vm.Table):
-    """My custom class."""
 
-    pass
+# This is applied properly as the final title color is taken from the fig.layout.template
+# (which is calculated from the current pio.template)
+pio.templates["vizro_dark"]["layout"]["title"]["font"]["color"] = "green"
+pio.templates["vizro_light"]["layout"]["title"]["font"]["color"] = "blue"
 
+# This doesn't work as expected as the final scatter points color is taken from fig.data
+# (which is calculated probably from the pio.templates.default)
+pio.templates["vizro_dark"]["layout"]["colorway"] = ["red"]
+pio.templates["vizro_light"]["layout"]["colorway"] = ["yellow"]
```

Comment:
I'm sad because there's no an easy way to adjust colour scales for dark/light themes for `px` charts... Should we explain the way to achieve even this (with `to_templated`) in the docs?